src/net/cgrand/enlive_html.clj

```diff
@@ -992,7 +992,7 @@
 (defn- nodify [node-spec]
   (cond
     (string? node-spec) node-spec
-    (vector? node-spec)
+    (and (vector? node-spec) (not (map? (first node-spec)))) 
```

Comment:
Been thinking and trying, but I can't figure out what you mean 8-°
That would give us:
```clj
(html (at (html [:a])
             [:a] (content "foo")))
=> ({:tag ::a, :attrs {}, :content ()})
```
which can't be what you want, or?